src/index.ts

```diff
@@ -1,30 +1,84 @@
 import { EventBridgeEvent } from 'aws-lambda';
+import { ItemType } from 'helpers/internal';
+import { EventBridgeSpy } from './spies';
 export * from './helpers';
 export * as matchers from './matchers';
 export * from './spies';
 export * from './utils/dynamodb';
 
+// Note: we cannot use the internal IMatcherHelperInput type here
+// because it does not work for some reason.
+type MatcherHelper<Name extends ItemType> = {
+  _slsJestHelperName: Name;
```

Comment:
I renamed this field to be more specific and avoid conflicts with other libraries or custom user types.
Otherwise there is a risk that non sls-jest types match the overload.